pyproject.toml

```diff
@@ -24,7 +24,7 @@ include = ["*.out", "*.yml"]
 python = ">=3.10,<3.14"
 numpy = "^2.0.0"
 scipy = "^1.15.2"
-sympy = "^1.13.3"
+sympy = ">=1.13.1"
```

Comment:
The `^` is just more restrictive than `>=`, because the meaning is exactly "greater equal, but less than next major".

In this case, it would be equivalent to `>=1.13.1,<2.0.0`.

So, I would keep the `^`